mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -526,6 +526,39 @@ func TestAccComposerEnvironment_ComposerV2(t *testing.T) {
 	})
 }
 
+func TestAccComposerEnvironment_ComposerV2HighResilience(t *testing.T) {
```

Comment:
I feel like we don't need a separate test for just one new field, I suggest incorporating it into different test (for example into TestAccComposerEnvironment_ComposerV2.